package.json

```diff
@@ -54,53 +51,20 @@
     "uglify-js": "^2.4.24",
     "vega": "^2.5.0",
     "vega-lite": "^1.0.1",
-    "watchify": "^3.2.3"
+    "webpack": "^1.12.14"
   },
   "scripts": {
-    "linkt": "npm run lint",
-    "lint:js": "jshint --verbose src/js/",
-    "vendor": "node ./scripts/vendor.js",
     "build": "npm run build:js && npm run build:scss",
-    "build:js": "browserify src/js/index.js -d -o vendor/js/lyra.js",
-    "postbuild:js": "uglifyjs vendor/js/lyra.js -cm > vendor/js/lyra.min.js",
-    "build:scss": "node-sass --include-path src/scss src/scss/main.scss vendor/css/main.css",
-    "postbuild:scss": "cleancss -o vendor/css/main.min.css vendor/css/main.css",
+    "build:js": "webpack -d",
+    "postbuild:js": "uglifyjs build/js/lyra.js -cm > build/js/lyra.min.js",
+    "build:scss": "node-sass --include-path src/scss src/scss/main.scss build/css/main.css",
```

Comment:
Long-term we may benefit from switching from explicitly calling UglifyJS to use Webpack production mode instead, but for now I opted to maintain the division of labor.